tests/e2e/macvtap_test.go

```diff
@@ -231,6 +234,16 @@ func filterPods(pods []v1.Pod, filterFunction func(v1.Pod) bool) []v1.Pod {
 	return filteredPods
 }
 
+func findMacvtapNamespace() string {
+	// Retrieve manager namespace serching for pods
+	// with label name=kubemacpool and getting the env var POD_NAMESPACE
```

Comment:
Remove this comment please; it adds nothing and mentions the wrong pod name.

Reply:
Done